src/components/BarChart/Axes/Axes.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+const Axes = memo(({ intervals }) => (
```

Comment:
Axes - plural
Axis - singular.

"BottomAxis" - is the best name for the component